clusters/l3-sqnc/capacity/sqnc-identity-service/values.yaml

```diff
@@ -0,0 +1,25 @@
+logLevel: info
+selfAddress: 5Gh5cFC3sPS7pK8SN41Ytp6VFsMyPpvKxtrrcLpkf19u9W83
+externalSqncNode:
+  host: magenta-sqnc-node-0
+  port: 9944
+authType: EXTERNAL
+externalOrigin: "https://capacity.dsch-l3.com"
+externalPathPrefix: "capacity/sqnc-identity-service"
```

Comment:
```suggestion
```
I think this can be removed as we are having the identity service and matchmaker APIs at the root of the subdomain.